rigging/connection.py

```diff
@@ -1,79 +1,136 @@
-# Copyright (C) 2023 Red Hat, Inc., Jake Hunsaker <jhunsake@redhat.com>
+import dbus
+import dbus.service
+import dbus.mainloop.glib
+from gi.repository import GLib
+from rigging.exceptions import (DBusServiceExistsError, 
+                            DBusServiceDoesntExistError,
+                            DBusMethodDoesntExistError)
 
-# This file is part of the rig project: https://github.com/TurboTurtle/rig
-#
-# This copyrighted material is made available to anyone wishing to use,
-# modify, copy, or redistribute it subject to the terms and conditions of
-# version 2 of the GNU General Public License.
-#
-# See the LICENSE file in the source distribution for further information.
```

Comment:
We need to keep these headers.

Reply:
Fixed in 93e685c868ab8eea455b5cc2f8af8cc95e75e0af